opengeode/AdaGenerator.py

```diff
@@ -434,9 +434,7 @@ def rec_detect_choice(expr: ogAST.Expression) -> bool:
             continue
         context_elems.append(f'{var_name.lower()} {type_name(var_type, False)}')
 
-    asn1_context = (f'\n{process_asn1}-Context ::='' SEQUENCE {\n'
-            + ",\n   ".join(line.replace("_", "-").replace("'", '"') for line in context_elems)
-            + "\n}\n")
+    asn1_context = ("\n{}-Context ::= SEQUENCE {}\n\n".format("hello", "\n,\n   ".join(line.replace("_", "-").replace("'", '"') for line in context_elems)))
```

Comment:
Why is this still there? I thought you had reverted it?

Reply:
I really apologize for this. I changed butt not added in this commit I added code of my test version!